adapter/config/ingress/status.go

```diff
@@ -68,14 +71,21 @@ func NewStatusSyncer(mesh *proxyconfig.ProxyMeshConfig, client kubernetes.Interf
 	if mesh.IngressService != "" {
 		publishService = fmt.Sprintf("%v/%v", options.Namespace, mesh.IngressService)
 	}
+	glog.Infof("INGRESS STATUS publishService ", publishService)
```

Comment:
This is one time at startup, and I think it is pretty important. The status IP of the ingress is calculated based on this.